### DISCUSSION TITLE
New Joomla/jdocs repo

### DISCUSSION THREAD

**Discussion by max123kl** [max123kl/J4.docs#13] [Announcements]
As @bembelimen has just announced to me, he will create a repo on GH for our purposes this weekend.

**Comment by max123kl** [comment]
@carlitorweb @ceford @alann60 
The question now is whether we want to transfer the previous content (or part of it) of this repo or whether we want to start from scratch

**Reply by ceford** [reply]
I think we should carry on here until we have a clear idea of what we are doing. The structure of the repo is an obvious thing to settle on before committing much content.

**Reply by carlitorweb** [reply]
@ceford exactly, structure of the docs will be important. I hope you with  @max123kl can define one. You two kown better the current directory at docs, so you kown better what work good and what not work for structure the docs. A good structure example can be the one you have at https://jdocmanual.org/

**Reply by max123kl** [reply]
Yes, the discussions here might be confusing for others on the Joomla repo.
But it is difficult to follow the discussions here properly.

I therefore suggest the following:
We open a separate thread for each topic or subject area. That way we can talk specifically about that topic and try not to go off-topic again and again.

**Reply by alann60** [reply]
Yes, carry on here until a structure is in place to a point where someone new could join and follow the workflow process.
